development/zh-cn/development-environment-setup.md

```diff
@@ -129,12 +129,11 @@ DolphinScheduler 的元数据存储在关系型数据库中，目前支持的关
 
 ##### 启动服务
 
-我们需要启动三个必须服务，包括 MasterServer，WorkerServer，ApiApplicationServer，如果有需求可以启动可选服务 LoggerServer
+我们需要启动三个服务，包括 MasterServer，WorkerServer，ApiApplicationServer，LoggerServer 已经被整合进了 MasterServer 和 WorkerServer，所以无需独立启动
```

Comment:
I think we should directly remove logger server in our document, because it unrelated to general user
```suggestion
我们需要启动三个服务，包括 MasterServer，WorkerServer，ApiApplicationServer
```